save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/cvsscalculator/ScoreCalculator.kt

```diff
@@ -0,0 +1,115 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.cvsscalculator
+
+import com.saveourtool.save.cvsscalculator.CvssMetrics.Companion.ATTACK_COMPLEXITY
+import com.saveourtool.save.cvsscalculator.CvssMetrics.Companion.ATTACK_VECTOR
+import com.saveourtool.save.cvsscalculator.CvssMetrics.Companion.AVAILABILITY
+import com.saveourtool.save.cvsscalculator.CvssMetrics.Companion.CONFIDENTIALITY
+import com.saveourtool.save.cvsscalculator.CvssMetrics.Companion.CVSS_VERSION
+import com.saveourtool.save.cvsscalculator.CvssMetrics.Companion.INTEGRITY
+import com.saveourtool.save.cvsscalculator.CvssMetrics.Companion.PRIVILEGES_REQUIRED
+import com.saveourtool.save.cvsscalculator.CvssMetrics.Companion.SCOPE
+import com.saveourtool.save.cvsscalculator.CvssMetrics.Companion.USER_INTERACTION
+import kotlin.math.min
+import kotlin.math.pow
+import kotlin.math.roundToInt
+
+@Suppress("UtilityClassWithPublicConstructor")
+private class CvssMetrics {
+    companion object {
+        const val ATTACK_COMPLEXITY = "/AC:"
```

Comment:
now it should be `AC` and etc

can we add a test for the calculator?

Reply:
Added test